mscore/exportxml.cpp

```diff
@@ -2314,9 +2314,9 @@ static int determineTupletNormalTicks(ChordRest const* const chord)
       if (!t)
             return 0;
       /*
-      qDebug("determineTupletNormalTicks t %p baselen %d", t, t->baseLen().ticks());
+      qDebug("determineTupletNormalTicks t %p baselen %s", t, qPrintable(t->baseLen().ticks().print()));
```

Comment:
you could also use `ticks().ticks()` here

Reply:
I actually prefer the fraction

Reply:
Me too.